Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -315,7 +315,7 @@ def set_learner(self, learner, key):
             # Removed
             self._invalidate([key])
             del self.learners[key]
-        else:
+        elif learner is not None:
```

Comment:
We haven't encountered this bug before because (I suppose) no learner widget can fail to produce a learner. Now it can - Calibrated Learner outputs `None` if it doesn't have a learner on its input.